openbb_terminal/portfolio/portfolio_controller.py

```diff
@@ -638,6 +643,112 @@ def call_alloc(self, other_args: List[str]):
                         f"are: {', '.join(self.AGGREGATION_METRICS)}"
                     )
 
+    @log_start_end(log=logger)
+    def call_attrib(self, other_args: List[str]):
+        """Process attrib command"""
+        parser = argparse.ArgumentParser(
+            add_help=False,
+            formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+            prog="attrib",
+            description="""
+                Displays sector attribution of the portfolio compared to the S&P 500            
+                """,
+        )
+        parser.add_argument(
+            "-p",
+            "--period",
+            type=str,
+            choices=portfolio_helper.PERIODS,
+            dest="period",
+            default="all",
+            help="The file to be loaded",
```

Comment:
Note this should be about the period.

Reply:
Hi Jeroen, can you please clarify what you mean by "about the period"? I believe this pattern was taken from call_perf(). (line 771 in portfolio_controller.py)

Reply:
The `help` argument says "The file to be loaded" while this relates to the period. So the description is wrong.